controller/script/destination-client/main.go

```diff
@@ -51,8 +51,9 @@ func main() {
 		switch updateType := update.Update.(type) {
 		case *pb.Update_Add:
 			log.Println("Add:")
+			log.Printf("metric_labels: %v", updateType.Add.MetricLabels)
 			for _, addr := range updateType.Add.Addrs {
-				log.Printf("- %s:%d", util.IPToString(addr.Addr.GetIp()), addr.Addr.Port)
+				log.Printf("- %s:%d - %v", util.IPToString(addr.Addr.GetIp()), addr.Addr.Port, addr.MetricLabels)
```

Comment:
note totally related to this change, but would you mind mentioning this tool in `BUILD.md`? maybe in the `Go` section, just something short like:

```bash
# forward localhost:8089 to the destination service
$ kubectl -n conduit port-forward $(kubectl --namespace=conduit get po --selector=conduit.io/control-plane-component=controller -o jsonpath='{.items[*].metadata.name}') 8089:8089

# test connecting to the destination service
$ bin/go-run  controller/script/destination-client/main.go -path stat-test-t2-svc.default.svc.cluster.local:90
```

also, at the top of this file, mind changing:
```go
addr := flag.String("addr", ":8089", "address of proxy api")
```
to:
```go
addr := flag.String("addr", ":8089", "address of destination service")
```

Reply:
@siggy wdyt https://github.com/runconduit/conduit/pull/654/commits/93a56b94f79e7f7342aee987119dc4d59b2362ed